mappings/net/minecraft/network/packet/s2c/play/SignedChatMessageS2CPacket.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/class_7438 net/minecraft/SignedChatMessageC2SPacket
```

Comment:
```suggestion
CLASS net/minecraft/class_7438 net/minecraft/network/packet/s2c/play/SignedChatMessageC2SPacket
```
Make sure you use enigma!

Reply:
```suggestion
CLASS net/minecraft/class_7438 net/minecraft/network/packet/s2c/play/SignedChatMessageS2CPacket
```
The class name also doesn't match the file nname.

Reply:
Ah, I wish we don't need to approve workflows... I mean, new contributors often make some faults that are easily examined by the build, and this approval requirement definitely does more inconvenience than good.